src/mlpack/tests/binarize_test.cpp

```diff
@@ -34,18 +32,18 @@ BOOST_AUTO_TEST_CASE(BinerizeOneDimension)
   const size_t dimension = 1;
   Binarize<double>(input, output, threshold, dimension);
 
-  BOOST_REQUIRE_CLOSE(output(0, 0), 1, 1e-5); // 1
-  BOOST_REQUIRE_CLOSE(output(0, 1), 2, 1e-5); // 2
-  BOOST_REQUIRE_CLOSE(output(0, 2), 3, 1e-5); // 3
-  BOOST_REQUIRE_SMALL(output(1, 0), 1e-5); // 4 target
-  BOOST_REQUIRE_SMALL(output(1, 1), 1e-5); // 5 target
-  BOOST_REQUIRE_CLOSE(output(1, 2), 1, 1e-5); // 6 target
-  BOOST_REQUIRE_CLOSE(output(2, 0), 7, 1e-5); // 7
-  BOOST_REQUIRE_CLOSE(output(2, 1), 8, 1e-5); // 8
-  BOOST_REQUIRE_CLOSE(output(2, 2), 9, 1e-5); // 9
+  REQUIRE(output(0, 0)== Approx(1.0).epsilon(1e-5 / 100)); // 1
+  REQUIRE(output(0, 1)== Approx(2.0).epsilon(1e-5 / 100)); // 2
+  REQUIRE(output(0, 2)== Approx(3.0).epsilon(1e-5 / 100)); // 3 
```

Comment:
```suggestion
  REQUIRE(output(0, 2)== Approx(3.0).epsilon(1e-5 / 100)); // 3
```

Remove the extra space at the end.

Reply:
Done